subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,80 @@
+"""
+Summary
+-------
+
+Operational Summary
+---------------------
+    1. Copy Unit file
+    2. Build image using predefined Dockerfile & p4321-server.py
+    3. Run image
+    4. Confirm p4321-server script is running
+    5. Stop container
+    6. cleanup
```

Comment:
Take a step back. Imagine that you don't know anything about this test. Does that description help you? "Copy unit file" -- what unit file? Copy where? What does it do? What would a future maintainer like to know about this test and how it works?